api/docs/release.dox

```diff
@@ -145,9 +145,6 @@ compatibility changes:
    expression and code relying on this needs to be rewritten.
```

Comment:
> Amends previous commit

Give the hash since it's not the actual previous commit.

Reply:
Done.